swarm.go

```diff
@@ -160,9 +178,18 @@ func (s *Swarm) teardown() error {
 	// Wait for everything to finish.
 	s.refs.Wait()
 
+	// close event bus emitters
+	_ = s.emitters.evtPeerConnectednessChanged.Close()
+
 	return nil
 }
 
+// EventBus is NOT a part of the network interface.
+// It has been exported to help with testing.
+func (s *Swarm) EventBus() event.Bus {
```

Comment:
doesn't seem like this needs to be exported. testing will either have the event bus it passed in to create the swarm, or can get it back from the host associated with the network.